website/sidebars.json

```diff
@@ -107,6 +107,7 @@
           "new-architecture-app-renderer-ios"
         ]
       },
+      "react-18-and-react-native",
```

Comment:
I will probably move this below the `"new-architecture-app-renderer-ios"`, in the `"Supporting in your App"` section.

Although it's true that the it also contains information on React/React-Native interaction, the steps we suggest are more related to Fabric and how to enable these features in the app.

Reply:
routing is not working

Reply:
Now that we split the page, would you still move it?

Reply:
Probably not. This is now something related to the new architecture in general but not a mandatory step when migrating an app.

Reply:
👍 Resolving this then